gtsfm/bundle/bundle_adjustment.py

```diff
@@ -178,23 +183,29 @@ def __calibration_priors(
         calibration_prior_noise_sigma = (
             CAM_CAL3FISHEYE_PRIOR_NOISE_SIGMA if is_fisheye_calibration else CAM_CAL3BUNDLER_PRIOR_NOISE_SIGMA
         )
-        if self._shared_calib:
+
+        for i in cameras_to_model:
+            camera = initial_data.get_camera(i)
+
+            calibration = None
```

Comment:
dont need this line?

Reply:
how come? Like does python work that way?